### DISCUSSION TITLE
Internal FTP access problem

### DISCUSSION THREAD

**Discussion by BertoMe** [AxisCommunications/discussions#219] [General]
I am trying to access a M7104 video server via ftp with an administrator user other than "root" but I can't because it tells me "access denied".

Can I access the Axis ftp server with a user other than root?

**Comment by vivekatoffice** [comment]
Related link:
https://help.axis.com/en-us/axis-os#changes-during-axis-os-11
![image](https://github.com/AxisCommunications/discussions/assets/52032125/8556d60c-6e73-470b-935c-cffb78b80dad)
